net-misc/icaclient/icaclient-22.7.0.20-r1.ebuild

```diff
@@ -180,39 +180,12 @@ src_install() {
 	insinto "${ICAROOT}"/keyboard
 	doins keyboard/*
 
+	cp -a util "${ED}/${ICAROOT}"
```

Comment:
Please add ```|| die```, since you are manually calling cp.

Reply:
thanks, changed